bin/stack_functions

```diff
@@ -400,11 +400,13 @@ function validate_radius_varz {
 
 function test_varz_value {
     varz=$1
-    if [ -z "$2" ]; then
-        port_path=8080/$2
+    port=$2
+    path=$3
+    if [ -z "$3" ]; then
+        query_path=0.0.0.0:$port
     else
-        port_path=8302
+        query_path=0.0.0.0:$port/$path
```

Comment:
If statement is likely unnecessary, since 0.0.0.0:8080 is equivalent to 0.0.0.0:8080/

Reply:
Addressed.